packages/@react-spectrum/textfield/src/TextFieldBase.tsx

```diff
@@ -98,7 +98,7 @@ export const TextFieldBase = forwardRef(function TextFieldBase(props: TextFieldB
   let validId = useId();
   let validationIcon = isInvalid
     ? <AlertMedium />
-    : <CheckmarkMedium id={validId} aria-label={stringFormatter.format('valid')} />;
+    : <CheckmarkMedium id={validId} aria-hidden aria-label={stringFormatter.format('valid')} />;
```

Comment:
Field will still announce valid due to the `aria-describedBy`, just preventing Talkback and NVDA from thinking the Checkmark should be focusable